### ISSUE TITLE
QS 1.5.6 (4021) crashes at boot (and other times? not sure)

### ISSUE THREAD

**Issue by fivetonsflax** [quicksilver/Quicksilver#2437]
On High Sierra 10.13.4 beta. I've been submitting reports every time, but I figure it can't hurt to open an issue instead. Any logs I can provide? Thanks.

**Comment by fivetonsflax** [comment]
A guess:

> 0 9:28:45 [pannonica:~] % grep -i quicksilver /var/log/system.log
Mar 26 09:03:28 pannonica com.apple.xpc.launchd[1] (com.apple.xpc.launchd.user.domain.501.100007.Aqua): failed lookup: name = com.blacktree.Quicksilver.ServiceProvider, handle = 0, instance = (n/a), flags = 0x1, requestor = Quicksilver.12504, error = 3
Mar 26 09:03:29 pannonica com.apple.xpc.launchd[1] (com.apple.xpc.launchd.user.domain.501.100007.Aqua): failed lookup: name = com.apple.decalog4.incoming, handle = 0, instance = (n/a), flags = 0x1, requestor = Quicksilver.12504, error = 3
Mar 26 09:03:29 pannonica com.apple.xpc.launchd[1] (com.apple.xpc.launchd.user.domain.501.100007.Aqua): failed activation: name = com.blacktree.Quicksilver.apple-extension-service, flags = 0x0, requestor = Quicksilver.12504, error = 1
Mar 26 09:03:30 pannonica Quicksilver[12504]: assertion failed: 17E193: libxpc.dylib + 75489 [25DB244E-217D-3CE0-A8F2-0C4255783B42]: 0x89
Mar 26 09:03:39 pannonica com.apple.xpc.launchd[1] (com.apple.xpc.launchd.user.domain.501.100007.Aqua): failed lookup: name = com.apple.AddressBook.AddressBookApplicationFrameworkIPC, handle = 0, instance = (n/a), flags = 0x1, requestor = Quicksilver.12504, error = 3
0 9:28:54 [pannonica:~] %

**Comment by fivetonsflax** [comment]
Just updated to 4024, I'll reopen if the crashes continue.

**Comment by fivetonsflax** [comment]
Still happening on 4024.

**Comment by skurfer** [comment]
Those messages don’t tell me much, but since your hostname is in your prompt, I was able to find some crash reports from your machine. I only found one from July and one recent one, and they weren’t the same cause.

For the more recent one, it might have something to do with scanning open applications, though it’s not clear exactly why. Are you able to reproduce it by launching a particular application, or switching between things very quickly, or something like that?
